unit-tests/rsutils/number/running-average/test-double.py

```diff
@@ -0,0 +1,45 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+from rspy import log, test
+from pyrsutils import running_average
+import random
+
+random.seed()
+
+
+def test_around( median, plus_minus, reps = 50, sets = 10 ):
```

Comment:
I would add some comment at the top of each test to say what it is you test.
The test goal.

test_around is not self explained..
Your call, I approved

Reply:
I don't know, there is no goal. I just tried various scenarios that would try to cover most cases: positive, negative, mix, small range, big range... I'll add something I guess.